src/main/java/com/github/alex1304/jdash/client/AbstractGDClient.java

```diff
@@ -438,6 +448,19 @@
 	public Mono<GDUserSearchData> getUserDataFromSearch(String searchQuery) {
 		return fetch(new GDUserSearchDataRequest(this, searchQuery, 0)).map(paginator -> paginator.asList().get(0));
 	}
+
+	/**
+	 * Gets comments of specific user in Geometry dash
+	 * @param accountId this account ID to get comments
+	 * @param page the page number
+	 * @return a Mono emitting a paginator containing all comments in user. Note that if
+	 *         no comments are found, it will emit an empty paginator. (In this case,
+	 *         fortunately, Geometry Dash API does not return the same response
+	 *         when an actual error occurs while processing the request).
+	 */
+	public Mono<GDPaginator<GDComment>> getUserCommentByAccountId(long accountId, int page){
```

Comment:
To be consistent with level comments, rename to `getCommentsForUser`

Reply:
Changed to _getCommentsForUser_